chromadb/test/test_cli.py

```diff
@@ -19,3 +29,86 @@ def test_app() -> None:
     )
     assert "chroma_test_data" in result.stdout
     assert "8001" in result.stdout
+
+
+def test_system_info() -> None:
+    result = runner.invoke(
+        app,
+        [
+            "env",
+            "info",
+        ],
+    )
+    assert "chroma_version" in result.stdout
+    assert "python_version" in result.stdout
+    assert "datetime" in result.stdout
+
+
+def test_system_info_with_remote() -> None:
+    try:
+        if (
+            requests.get(
+                f"http://localhost:{os.environ.get('CHROMA_SERVER_HTTP_PORT', 8000)}/api/v1/heartbeat"
+            ).status_code
+            != 200
```

Comment:
this gives me the heebie-jeebies, to the point where i think i'd rather not have this test. @HammadB wdyt?

Reply:
Changed to use `CHROMA_INTEGRATION_TEST` flag.